config/samples/kubectl-maestro

```diff
@@ -374,6 +393,96 @@ then
     fi
 fi
 
+function createFunction {
+    framework=$2
+    shift 2
+    while getopts ":n:v:p:" opt ; do
+        case "$opt" in
+                p)   
+                c+=("$OPTARG")
+                ;;
+                n)
+                echo "name is: $OPTARG"
+                name=$OPTARG
+                ;;
+                v)
+                echo "version is: $OPTARG"
+                frameworkversion=$OPTARG
+                ;;                
+                \?)
+                echo "Invalid option: -$OPTARG" >&2
+                exit 1
+                ;;
+                :)
+                echo "Option -$OPTARG requires an argument." >&2
+                exit 1
+                ;;
+        esac
+    done
+    shift $((OPTIND-1))
+    rm /tmp/maestro_instance_parameter
+    replace=": "
+    for t in ${c[@]}; do
+        param_clean=${t/:/$replace}
+        cat <<EOF >> "/tmp/maestro_instance_parameter"
+        $param_clean
+EOF
+    done
+    cat <<EOF > "/tmp/maestro_instance"
+apiVersion: maestro.k8s.io/v1alpha1
+kind: Instance
+metadata:
+    labels:
+        controller-tools.k8s.io: "1.0"
+        framework: $framework
+    name: $name
+spec:
+    frameworkVersion:
+        name: $framework-$frameworkversion
+        namespace: default
+        type: FrameworkVersion
+    parameters:
+EOF
+    cat /tmp/maestro_instance /tmp/maestro_instance_parameter > /tmp/maestro_instance_joined
+    kubectl apply -f /tmp/maestro_instance_joined
+}
+
+# create argument handling
+if [[ "$1" == "create" ]]
+then
+    createFunction "$@"
+    if [[ "$?" == "1" ]]
+    then
+        echo ""
+        echo "Failed to create instance for framework $2."
+        echo ""
+        echo "Invalid create command."
+        echo ""
+        echo "Syntax: kubectl maestro create -n <instanceName> -v <versionNumber> -p PARAM1=VALUE -p PARAM2=Value"
+        echo ""
+        echo "Example: kubectl maestro create kafka -n kafka -v 2.11-2.4.0 -p KAFKA_ZOOKEEPER_URI:zk-zk-0.zk-hs:2181,zk-zk-1.zk-hs:2181,zk-zk-2.zk-hs:2181 -p KAFKA_ZOOKEEPER_PATH:/small -p BROKERS_COUNT:3"
```

Comment:
This actually won't work and results in:

```
ERROR: logging before flag.Parse: E0125 18:12:40.665187   34488 reflector.go:205] github.com/maestrosdk/maestro/vendor/sigs.k8s.io/controller-runtime/pkg/cache/internal/informers_map.go:126: Failed to list *v1alpha1.Instance: v1alpha1.InstanceList.ListMeta: v1.ListMeta.TypeMeta: Kind: Items: []v1alpha1.Instance: v1alpha1.Instance.Spec: v1alpha1.InstanceSpec.Parameters: ReadString: expects " or n, but found 3, error found in #10 byte of ...|S_COUNT":3,"KAFKA_ZO|..., bigger context ...|FrameworkVersion"},"parameters":{"BROKERS_COUNT":3,"KAFKA_ZOOKEEPER_PATH":"/small","KAFKA_ZOOKEEPER_|...
```

Not sure about why this error insists of " and if we could find a way around @runyontr ?

Reply:
I typically have to escape the integer values since the parameters section is a `map[string]string.  So `BROKERS_COUNT: \"3\"` or the like.